### PR TITLE
Make Dapps great again

### PR DESCRIPTION
**Pull request by flexsurfer**

### CODE REVIEW

File: src/status_im/ui/screens/home/views/inner_item.cljs [hunk 1]
```diff
@@ -99,13 +99,13 @@
           [message-content-text last-message]
           [unviewed-indicator chat-id]]]]])))
 
-(defview home-list-browser-item-inner-view [{:keys [browser-id name url dapp? contact] :as browser}]
-  (letsubs [contact' [:get-contact-by-identity contact]]
```
Comment (jeluard) [inline src/status_im/ui/screens/home/views/inner_item.cljs:103]: Yeahh!!

**Review by jeluard — APPROVED** [review]

**Comment by goranjovic** [comment]
This PR doesn't have its issue?

**Review by goranjovic — APPROVED** [review]

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/V3Z995 for branch PR-4452

<blockquote><div><strong><a href="https://i.diawi.com/V3Z995">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/sD6Apa for branch PR-4452

<blockquote><div><strong><a href="https://i.diawi.com/sD6Apa">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by asemiankevich** [comment]
@flexsurfer when tapping Asset in Simple Dapp
image

**Comment by flexsurfer** [comment]
@asemiankevich not related to this PR

**Comment by goranjovic** [comment]
@flexsurfer @asemiankevich Just merged the fix for that in develop. If you rebase it should be gone

**Pull request merged**